docs/modules/user-guide/partials/proc_adding-a-registry-schema.adoc

```diff
@@ -0,0 +1,423 @@
+[id="adding-a-registry-schema_{context}"]
+= Adding a registry schema
+
+[role="_abstract"]
+To add samples and stacks from other repositories, put the file, `extraDevfileEntries.yaml`, under the root of a devfile registry. This file contains the location information of the extra samples and stacks.
+
+.extraDevfileEntries.yaml schema
+* samples: schema[] - the sample list
+* stacks: schema[] - the stack list Stack/Sample attributes:
```

Comment:
This looks like it's malformed. Is "Stack/Sample attributes" supposed to be a header for the next series of definitions?